### PR TITLE
gitlight 0.17.5 (new cask)

### PR DESCRIPTION
**Pull request by Hacksore**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by Hacksore** [comment]
FYI @colinlienard 🤝

File: Casks/g/gitlight.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask "gitlight" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "0.17.5"
+  sha256 arm:   "11cf26a907520b734b009b91578240b750c7210f8e498fc742c4eca0b87c5995",
+         intel: "fd066caa44b37fd81619ca1e3ed9fb3465b00cfd3bddbb938d88d72162805890"
+
+  url "https://github.com/colinlienard/gitlight/releases/download/gitlight-v#{version}/GitLight_#{version}_#{arch}.dmg",
+      verified: "github.com/colinlienard/gitlight/"
+  name "gitlight"
```
Comment (khipp) [inline Casks/g/gitlight.rb:10]:
```suggestion
  name "GitLight"
```

File: Casks/g/gitlight.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  desc "GitHub & GitLab notifications on your desktop"
```
Comment (khipp) [inline Casks/g/gitlight.rb:11]: We should rewrite the description to not use user pronouns (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)). Possible alternatives could be:

```suggestion
  desc "GitHub & GitLab desktop notifications"
```

or

```suggestion
  desc "Desktop notifications for GitHub & GitLab"
```

**Review by khipp — COMMENTED** [review]
Thank you for your submission, @Hacksore!

**Review by khipp — APPROVED** [review]
Thank you, @Hacksore!

**Pull request merged**